src/huggingface_hub/repocard.py

```diff
@@ -79,7 +79,22 @@ def __init__(self, content: str):
 
         </Tip>
         """
+
+        # Set the content of the RepoCard, as well as underlying .data and .text attributes.
+        # See the `content` property setter for more details.
         self.content = content
+
+    @property
+    def content(self):
+        """The content of the RepoCard, including the YAML block and the Markdown body."""
+        line_break = _detect_line_ending(self._content) or "\n"
+        return f"---{line_break}{self.data.to_yaml(line_break=line_break)}{line_break}---{line_break}{self.text}"
+
+    @content.setter
+    def content(self, content: str):
+        """Set the content of the RepoCard."""
+        self._content = content
```

Comment:
I think this private attribute is not used, right ?

Reply:
ah yes you're right. I originally had updated to use this in `__init__` but copy pasted everything down to this setter method later. It's useless and can go. will patch real quick. 

Nice catch!! thanks

Reply:
EDIT - I lied. it's being used in the `content` property.